cpu/efm32/families/efm32lg/cpus.txt

```diff
@@ -0,0 +1,72 @@
+# This file is automatically generated, and should not be changed. There is
+# propbably little reason to edit this file anyway, since it should already
```

Comment:
I can deduce the typo in 'probably' is also automatically generated :)

Reply:
Addressed in #8618.